src/program.jl

```diff
@@ -77,16 +77,15 @@ function Program(ctx::Context; source=nothing, binaries=nothing)
 end
 
 #TODO: build callback...
-function build!(p::Program; options="", raise=true)
+function build!(p::Program; options = "", raise = true)
     opts = String(options)
     ndevices = 0
     device_ids = C_NULL
-    @check api.clBuildProgram(p.id, cl_uint(ndevices), device_ids, opts, C_NULL, C_NULL)
```

Comment:
because it throws, when the compilation fails :( we'd need a more fine grained control, but as it only checks if compilation succeeds and we check it with `info`, we might be fine here!